intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -428,29 +432,32 @@ \subsection{Generic resource requirements}
 \end{prompt}
 
 For the simplest cases, only the amount of maximum estimated execution time
-(called ``walltime'') is really important. Here, the job will not require more
-than 2 hours, 30 minutes to complete. As soon as the job would take more time,
+(called ``walltime'') is really important. Here, the job requests a reservation of
```

Comment:
not sure if the term 'reservation' should be used here, since that has a very specific meaning?

Reply:
You have a point here, I will remove it.